Mise a jour contenu du site vitrine pour le salon de Lyon

etienneCharignon opened the pull request:
fix  betagouv/competences-pro#597

![Capture d’écran 2019-12-06 à 13 58 14](https://user-images.githubusercontent.com/298214/70324703-7c3b9d80-1830-11ea-82db-77d9832515e6.png)

etienneCharignon committed:
Met à jour le block titre et crée et met à jour le block objectif

etienneCharignon committed:
Ajoute le bouton vers l'application

etienneCharignon committed:
Ajoute la page "qui sommes nous ?"

etienneCharignon committed:
évolution de "compétences" vers "compétences évaluées"

etienneCharignon committed:
Ajoute le bouton "acceder à eva" dans le menu"

morissetcl commented on layouts/_default/baseof.html:
```diff
@@ -52,6 +52,7 @@ <h1 class="col-2"><a href="/"><img src="/eva.svg" alt="eva" width="73px"></a></h
             {{ range .Site.Menus.main }}
               <a class="ml-5 font-weight-bold lien-menu {{ if $currentPage.IsMenuCurrent "main" . }}actif{{ end }}" href="{{ .URL }}">{{ .Name }}</a>
             {{ end }}
+              <a class="btn btn-primary btn text-center ml-5 pl-4 pr-4" href="//app.eva.beta.gouv.fr">Accéder à eva</a>
```
A mettre dans une trad comme l'email de contact par ex ?

etienneCharignon replied:
fait

etienneCharignon committed:
Améliore l'affichage sur petit écran.

- le bouton "accéder à eva" se redimentionne pour prendre plus de place sur les petits écrans
- le menu "qui sommes-nous ?" ne peut plus se couper en morceau (entièrement unbreakable)

shanser reviewed (approved):
👏
